packages/platform/examples/base/src/pages/booking.tsx

```diff
@@ -54,6 +59,11 @@ export default function Bookings(props: { calUsername: string; calEmail: string
               setBookingTitle(data.data.title);
               router.push(`/${data.data.uid}`);
             }}
+            duration={eventTypeDuration}
+            entity={{
+              orgSlug: "ecorp",
+              considerUnpublished: false,
+            }}
```

Comment:
I am wondering what are the potential risk of enabling users to control their orgSlug